draft-ietf-jsonpath-base.md

```diff
@@ -1311,6 +1306,22 @@ JSONPath:
   addressed by the previous path fragment. Array indices always start
   at 0.
 
+# JSON Pointer
+
+This appendix is informative.
+
+JSONPath is not intended as a replacement for, but as a more powerful
+companion to, JSON Pointer {{RFC6901}}. The purposes of the two standards
+are different.
+
+JSON Pointer is for identifying a single value within a JSON document whose
+structure is known.
+
+JSONPath can identify a single value within a JSON document, for example by
+using a Normalized Path. But JSONPath is also a query syntax that can be used
+to search for and extract multiple values from JSON documents whose structure
+is partially known.
+
```

Comment:
I think we should say something about conversion between JSON Pointer and JSONPath.

```
A Normalized JSONPath can be converted into a JSON Pointer by converting the syntax.
The inverse is not generally possible as a path component in a JSON Pointer that looks like a number may identify a member of a JSON object or may index an array.  For this conversion, knowledge of the structure of the references JSON value is needed.